src/factoids.coffee

```diff
@@ -43,22 +43,22 @@ module.exports = (robot) ->
       to ?= msg.message.user.name
       msg.send "#{to.trim()}: #{fact.value}"
 
-  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{2,}\\w)", 'i'), (msg) =>
+  robot.respond new RegExp("[#{prefix}]([\\w\\s-]{1,}\\w)", 'i'), (msg) =>
```

Comment:
Yeah, I think I overlooked the last \w. Hmm. I'm not even sure I intended for it to match spaces. Can't recall why the regexp is written that way. I think we can safely remove the `\\s` while we're at it, what do you think? 

This will probably need to be a major ver bump, though, since it might break someone's database (slim chance, but still).